geolib/models/dsheetpiling/settings.py

```diff
@@ -34,7 +34,7 @@ class ModelType(IntEnum):
     """Represents the model types from D-Sheet Piling using the same integer values used in the application.
 
     The Model type determines the kind of construction: single pile or wall.
-    Note that the model type WALL includes sheet piling, wooden sheet piling and diaphragm wall.
+    Note that the model type SHEET_PILING includes sheet piling, wooden sheet piling and diaphragm wall.
     """
 
     WALL = 0
```

Comment:
it is not changed in settings.py (Note: build on TC fails) and other places e.g. internal.py